modules/gitops-release-notes-1-4-6.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-4-6_{context}"]
+= Release notes for {gitops-title} 1.4.6
+
+[role="_abstract"]
+{gitops-title} 1.4.6 is now available on {product-title} 4.7, 4.8, 4.9, 4.10, and 4.11.
```

Comment:
I'm sorry, my bad. OCP 4.11 is not released yet. Maybe we shouldn't mention it in our release notes?